caffe2/python/helpers/normalization.py

```diff
@@ -5,12 +5,15 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-from caffe2.python import core
+from caffe2.python import core, scope
 
 
 def lrn(model, blob_in, blob_out, order="NCHW", use_cudnn=False, **kwargs):
     """LRN"""
-    if use_cudnn:
+    dev = kwargs['device_option'] if 'device_option' in kwargs \
+        else scope.CurrentDeviceScope()
+    is_cpu = dev is None or dev.device_type == caffe2_pb2.CPU
```

Comment:
actually our automatic linter noticed that problem. Weird that it works for you...!